include/common_package/commands.hpp

```diff
@@ -346,10 +346,7 @@ class CommandDefinitions {
               MAX_FLIGHT_HEIGHT_CM}},
             {"target_height_cm",
              {true, number_unsigned, 0, MAX_FLIGHT_HEIGHT_CM}},
-            {"horizontal_speed_mps",
-             {true, number, 0.0, MAX_HORIZONTAL_SPEED_MPS}},
-            {"vertical_speed_mps",
-             {true, number, 0.0, MAX_VERTICAL_SPEED_MPS}}};
+            {"speed_mps", {true, number, 0.0, MAX_HORIZONTAL_SPEED_MPS}}};
```

Comment:
Is MAX_VERTICAL_SPEED_MPS unused now? Why is MAX_HORIZONTAL_SPEED_MPS used for "speed_mps"?

Reply:
This is what I wrote into the Dev Guide:

As the FCC can not differentiate between the horizontal and vertical speed restrictions, the following rules are defined:
- The horizontal speed restriction shall always be used when the main way of movement is horizontal (e.g. during flight to waypoint).
- The vertical speed restriction shall always be used when the main way of movement is vertical (e.g. during takeoff and landing).

Reply:
But as I'm implementing the Waypoint node right now, I figured I have to return those values so nevermind

Reply:
I am a bit confused as to what is goind on now. So you are currently not using the MAX_VERTICAL_SPEED_MPS, but that is only temporarily the case. Furthermore you are using the MAX_HORIZONTAL_SPEED_MPS for the a overall speed value for now, also only temporarily. Did I get that right?

Reply:
I'm sorry, my message was confusing.
I ment that I will revert the changes in this commit and add the separation back in.